app/views/posts/index.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Here is a list of all the posts</h1>
\ No newline at end of file
+<body>
+    <%= render "partials/user_container", user: @users %>
+    <% @posts.each do |post| %>
+    <h3>Post Title: <%= @posts.title %></h3>
+    <p><%= @posts.text %></p>
+    <div class="block">
+        <p>Comments: <%= @posts.comments_counter %></p>
+        <p>Likes: <%= @posts.likes_counter %></p>
+    </div>
+    <% end %>
+    <button>Pagination</button>
+</body>
```

Comment:
- [ ] there is an error in your post index, kindly fix this
![Screenshot from 2023-04-21 17-14-02](https://user-images.githubusercontent.com/66097371/233685124-26735c4d-e331-432a-bc49-2d768f46fc92.png)